src/components/TheHeader.vue

```diff
@@ -2,7 +2,7 @@
   <header>
     <v-toolbar data-cy="nav-drawer" :color="variant === 'light' ? 'surface-2' : 'surface-1'">
       <v-toolbar-title>
-        <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
```

Comment:
From what i experimented with, the <v-btn> component automatically applies cursor pointer, so it was redundant

Reply:
@IbrahimKhanGH that makes sense; good catch!

Btw one really handy thing you can do when submitting a PR is add PR comments like this one preemptively to explain anything that's not immediately obvious. Things that aren't necessarily "in scope" for an issue but were quick fix like this can be a good fit for a quick comment. Same goes for anything that you expect the reviewer might read and ask "why are they changing this?"

PR comments can be a helpful supplement to code comments when the thing you want to explain isn't about the end result of the code but rather the motivation for the change. Like in this case, there wouldn't be a need to see  any comment in the code after this is merged since we're actually removing the style from this element so there is nothing to comment on. But the change itself is justified by just what you noticed: this style is redundant. That's a good reason for a quick PR note so the reviewer can get quickly up to speed. It also makes for a good chance for you to review your own changes to catch anything that is included by accident and to generally make sure you're PRing exactly the changes you intend.

None of this is a criticism, I just figured that since you mentioned this being your first open source PR that you might like to hear that it's okay to comment on your own PR preemptively and that it can be helpful to do so.

Again, good catch on this extraneous style; we do want to remove those. If it turned out to be a bigger issue changing lots of files to clean something else up, then it would make sense to address it in a separate issue + PR. But since it's just a one line fix, it's fine to include it in your existing one, just as you've done.

Reply:
No worries, I want all the feedback I can get. Im completely new to open source, and want to get better at larger scale development, so anything that can teach me to improve is something I want. So, Thank you for all the reviews and the support through my first contribution🙏 I really appreciate it and will be contributing more in the future